modules/serverless-ossm-setup.adoc

```diff
@@ -19,14 +19,20 @@ ifdef::openshift-dedicated,openshift-rosa[]
 endif::[]
 
 * You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
-* Install {SMProductName}. {ServerlessProductName} with {SMProductShortName} only is supported for use with {SMProductName} version 2.0.5 or later.
+
+* Install the {SMProductName} Operator and create a `ServiceMeshControlPlane` resource in the `istio-system` namespace. If you want to use mTLS functionality, you must also set the `.spec.security.dataPlane.mtls` set to `true`.
```

Comment:
```suggestion
* Install the {SMProductName} Operator and create a `ServiceMeshControlPlane` resource in the `istio-system` namespace. To use mTLS functionality, you must also set the `.spec.security.dataPlane.mtls` field to `true`.
```

Reply:
Half accepted - the "field" addition I think is good (seems like it was a typo previously), but keeping "If you want to..." to retain the active voice etc. per docs guidelines.